.github/copilot-instructions.md

```diff
@@ -0,0 +1,316 @@
+# OpenProject Coding Agent Instructions
+
+## Repository Overview
+
+**OpenProject** is a web-based, open-source project management software written in Ruby on Rails. It uses PostgreSQL for data persistence and supports features like project planning, task management, Agile/Scrum, time tracking, wikis, and forums.
+
+- **Size**: Large monorepo (~1M+ lines of code)
```

Comment:
How old is the repo?
How big is the repo in MB?

N.B. OpenProject was originally a fork of Redmine, so it's possible there are vestiges of Redmine code in the codebase. However the project was forked well over a decade ago (@lindenthal - verify?) and has evolved significantly in its own right.